### PR TITLE
Step 1 jenny

### PR DESCRIPTION
**Pull request by jiayi-jenny-zhou**
General things:

What is the purpose of this PR?
Learn how to write new code and work pybind

What changes did you make? Why?
 I made unit tests for pricingutils then implemented pricingtutils

What bugs did you find while testing?
it got mad at me for not having an explicit constructor

This PR Specific:

What challenges did you face while writing the module from scratch?
mathing correctly for the tests

How did you ensure your unit tests are comprehensive?
spammed zero negative positive decimal

Did you have enough guidance to complete the task?
yes

### CODE REVIEW

File: src/pricingutil.cpp [hunk 1]
```diff
@@ -1 +1,12 @@
 #include "pricingutil.h"
+
+PricingUtil::PricingUtil(){}
```
Comment (minnce) [inline src/pricingutil.cpp]: Initialize val here please

**Review by minnce — CHANGES_REQUESTED** [review]

File: src/pricingutil.cpp (hunk 1 above)
Comment (jiayi-jenny-zhou) [reply src/pricingutil.cpp]: like this?

Comment (minnce) [reply src/pricingutil.cpp]: yup perfect

**Review by minnce — APPROVED** [review]
Nice work, thanks for making the change!

**Pull request merged**